src/elements/common/item/ItemSubDetails.scss

```diff
@@ -8,7 +8,8 @@
     }
 }
 
-.bce-item-row {
+.bce-item-row,
+.bcp-item-row {
```

Comment:
Is this dependent on `.bce-item-row` at all? Could the extra specificity be dropped?

Reply:
@ConradJChan Yup, without the extra specificity, the grid view also displays a middot:
<img width="268" alt="content-explorer-with-middot-spacing" src="https://user-images.githubusercontent.com/2956895/115625360-d071e100-a2b0-11eb-8202-8b2e50e9d8a7.png">

Reply:
Ah missed the duality of grid vs list, thanks